protocols/s3/selectel.md

```diff
@@ -0,0 +1,47 @@
+Selectel S3 Storage
+====
+
+```{image} _images/selectel.png
+:alt: Selectel Drive Icon
+:width: 128px
+```
+
+> [Selectel Cloud Storage](https://docs.selectel.ru/en/cloud/object-storage/) is a durable S3 for developers and businesses.
+
+## Connecting
+
+The integration Cyberduck with Selectel can be operated through a graphical or console client via the S3 API.
```

Comment:
Not sure what you mean by `…can be operated through a graphical or console client via the S3 API.`.